src/labels/CustomLabels.labels

```diff
@@ -1729,6 +1729,14 @@
         <shortDescription>Checks if an Elevate Recurring Donation&apos;s amount is valid or not</shortDescription>
         <value>Please enter a valid Amount. The Amount cannot be less than 1.</value>
     </labels>
+    <labels>
```

Comment:
@sam-knox can you review all the new custom labels. All of them are specific to each validation scenario.